src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx

```diff
@@ -50,9 +50,23 @@ To add a Lambda layer to your function, follow these steps:
      },
    });
    ```
-
-   The Lambda layer is represented by an object of key/value pairs where the key is the module name that is exported from your layer and the value is the ARN of the layer. The key (module name) is used to externalize the module dependency so it doesn't get bundled with your lambda function. A maximum of 5 layers can be attached to a function, and they must be in the same region as the function.
    
+   The Lambda layer is represented by an object of key/value pairs where the key is the module name that is exported from your layer and the value is the ARN of the layer. The key (module name) is used to externalize the module dependency so it doesn't get bundled with your lambda function. A maximum of 5 layers can be attached to a function, and they must be in the same region as the function.
+   <br/><br/>Alternatively, you can specify the layer as `myLayer:1` where `myLayer` is the name of the layer and `1` is the version of the layer. For example:
+
+   ```ts title="amplify/functions/my-function/resource.ts"
+   import { defineFunction } from "@aws-amplify/backend";
+
+   export const myFunction = defineFunction({
+     name: "my-function",
+     layers: {
+      "module": "myLayer:1"
```

Comment:
```suggestion
      "some-module": "myLayer:1"
```

module -> some-module to mitigate confusion with node's `module`, and to make it clearer that this can be any module